remoteappmanager/db/orm.py

```diff
@@ -1,8 +1,10 @@
 import contextlib
+import sqlite3
```

Comment:
Keep it encapsulated in the enabling method, so that it works regardless if we have/use sqlite or not.